libraries/libfc/src/crypto/bls_private_key.cpp

```diff
@@ -13,6 +13,12 @@ namespace fc::crypto::blslib {
       return bls_public_key(pk);
    }
 
+   bls_signature bls_private_key::pop_proof() const
+   {
+      bls12_381::g2 proof = bls12_381::pop_prove(_sk);

```

Comment:
As @dimas1185 mentioned, the bls12-381 submodule should be updated to use the following commit: https://github.com/AntelopeIO/bls12-381/commits/5b18ef4f8add17d17aa8fe9233aa363829aca90a. Then this call to `pop_prove` would generate the correct signature.

Reply:
Also, I think renaming it would be better to rename the function to either `proof_of_possession()` or, if we want to keep it short, `generate_pop()`. 

@heifner, it seems like you would prefer `proof_of_possession()`?

Reply:
Yes, I think `pop` is not necessarily immediately clear.

Reply:
`pop` says nothing to general public. I support using `proof_of_possession`

Reply:
I also tested `leap-util` generated keys with requested bls commit and it works. I have changes that are ready for PR. Waiting for Lin to push this.

Reply:
I will change it to `proof_of_possession()` and update the submodule.